static/app/types/system.tsx

```diff
@@ -228,17 +228,38 @@ export type PipelineInitialData = {
   props: Record<string, any>;
 };
 
-export type Broadcast = {
-  cta: string;
+export interface Broadcast {
   dateCreated: string;
   dateExpires: string;
+  /**
+   * Has the item been seen? affects the styling of the panel item
+   */
   hasSeen: boolean;
   id: string;
   isActive: boolean;
+  /**
+   * The URL to use for the CTA
+   */
   link: string;
+  /**
+   * A message with muted styling which appears above the children content
+   */
   message: string;
   title: string;
-};
+  /**
+   * Category of the broadcast.
+   * Synced with https://github.com/getsentry/sentry/blob/923a65508912c3e181e1c70cbdf076b7b956aa90/src/sentry/models/broadcast.py#L14
```

Comment:
can we change the branch here to be master: https://github.com/getsentry/sentry/blob/master/src/sentry/models/broadcast.py

Reply:
I tried that a couple of times, but always when I clicked on the copy button I got a `blob/code`